code/datums/mutations/_mutations.dm

```diff
@@ -93,7 +93,7 @@
 	return
 
 /datum/mutation/human/proc/on_losing(mob/living/carbon/human/owner)
-	if(owner && istype(owner) && (owner.dna.mutations.Remove(src)))
+	if(!owner || !istype(owner) || !(owner.dna.mutations.Remove(src)))
```

Comment:
no need for !owner, istype does it already

Reply:
will do